src/BotwinExtensions.cs

```diff
@@ -12,11 +12,20 @@ namespace Botwin
 
     public static class BotwinExtensions
     {
+        /// <summary>
+        /// Adds Botwin to the specified <see cref="IApplicationBuilder"/> with the specified <see cref="IRouter"/>.
+        /// </summary>
+        /// <returns>A reference to this instance after the operation has completed.</returns>
         public static IApplicationBuilder UseBotwin(this IApplicationBuilder builder)
         {
             return UseBotwin(builder, null);
         }
 
+        /// <summary>
+        /// Adds Botwin to the specified <see cref="IApplicationBuilder"/> with the specified <see cref="IRouter"/>.
+        /// </summary>
+        /// <param name="options">BotwinOptions <see cref="BotwinOptions"/> with global before and after hooks.</param>
```

Comment:
Probably best to remove `with global before and after hooks` in case we add more to it and forget to update this xml 😄